packages/datetime/src/common/months.ts

```diff
@@ -4,17 +4,23 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
+/**
+ * Enumeration of calendar months.
+ *
+ * Note that the enum values are numbers (with January as `0`) so they can be
+ * easily compared to `date.getMonth()`.
+ */
 export const enum Months {
-    JANUARY,
-    FEBRUARY,
-    MARCH,
-    APRIL,
-    MAY,
-    JUNE,
-    JULY,
-    AUGUST,
-    SEPTEMBER,
-    OCTOBER,
-    NOVEMBER,
-    DECEMBER,
+    JANUARY = 0,
```

Comment:
Why numbers and not strings?

Reply:
So, take a look at the comment. It explicitly mentions comparing with Date.getMonth() which uses zero-indexed months.

Reply:
Oh, whoops, missed the comment. Good good.